addons/im_livechat/static/src/embed/common/thread_patch.xml

```diff
@@ -6,7 +6,7 @@
                 <Message message="props.thread.livechatWelcomeMessage" hasActions="false" thread="props.thread" className="'px-3'"/>
             </div>
         </xpath>
-        <xpath expr="//*[@name='content']" position="after">
+        <xpath expr="(//div[hasclass('o-mail-Thread')])[last()]" position="inside">
             <Message t-if="props.thread.chatbot?.typingMessage" message="props.thread.chatbot.typingMessage" hasActions="false" isInChatWindow="env.inChatWindow" isTypingMessage="true"  thread="props.thread"/>
```

Comment:
maybe the fix should be to make this `elif` rather than `if`?

Reply:
The inserted element is the `typing message`. I think it makes more sense to keep it at the end of the thread rather than check it in the middle of `t-if/t-else` with `t-elif`. Although both of them work functionally.